Mobile Buy SDK Sample Apps/Advanced App - ObjC/Mobile Buy SDK Advanced Sample/ProductListViewController.h

```diff
@@ -27,6 +27,10 @@
 @import UIKit;
 @import Buy;
 
+//#warning - Enter your merchant ID
+// Adding a merchant ID will show Apple Pay in the BUYProductViewController (on supported devices)
+#define MERCHANT_ID @""
```

Comment:
At this point, the credentials (api key, domain, merchant id) should be in a seperate file, shared by apps and extensions